tests/behave/features/steps/legal_adviser.py

```diff
@@ -0,0 +1,28 @@
+from behave import given, when, then
+from features.helpers.fala import get_fala_url
+
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+
+
+@given("I am a on the fala landing page")
+def step_fala_landing_page(context):
+    get_fala_url(context, "/")
+
+    WebDriverWait(context.driver, 10).until(EC.presence_of_element_located((By.ID, "content")))
+
+
+@when(u'I search for a legal adviser with postcode "{postcode}" and category "{category}"')
+def step_search_legal_adviser(context, postcode, category):
+    context.driver.find_element_by_xpath("//input[@type='text'][@name='postcode']").send_keys(postcode)
+    context.driver.find_element_by_xpath("//input[@name='categories'][@value='{}']".format(category))
+    context.driver.find_element_by_id("searchButton").click()
+    WebDriverWait(context.driver, 10).until(EC.presence_of_element_located((By.CLASS_NAME, "legal-adviser-results")))
+
+
+@then(u'I should see a list of legal advisers')
+def step_impl(context):
+    if not len(context.driver.find_elements_by_xpath("//div[@class='search-results-list']/ul[@class='org-list']/li")):
+        raise Exception("Could not find list of legal advisers")
```

Comment:
:raising_hand_man: In feature tests, it's typically useful to use assertions like in unit tests; it gives a richer context to the failure and usually conveys the intent better.

```python
results_of_the_search = context.driver.find_elements_by_xpath("//div[@class='search-results-list']/ul[@class='org-list']/li")
assert_greater(results_of_the_search, 0)
```

The other test is using `assert_equal` for this effect (the assertion fail would print out the diff).